data/human/hails.txt

```diff
@@ -1708,6 +1708,85 @@ phrase "friendly civilian"
 		", but I still had to take a big loan to buy my first ship."
 		", but it was my grandma's inheritance that allowed me to buy this ship."
 
+phrase "friendly civilian"
+	word
+		"Don't be too picky about the jobs you take. It's better to"
+	word
+		" "
+	word
+		"ship crooks, than have a crooked ship."
+		"ship slugs, than have a sluggish ship."
+		"ship bad bands, than have your ship badly banned."
+		"be working hard at shipping, than have your ship hardly working."
+
+
+phrase "friendly civilian"
+	word
+		"Ey Comrade,"
+	word
+		" "
+	word
+		"your ship has loaded on a planet? I have loaded a planet onto my ship."
+		"you have cargo in your ship? I have ships in my cargo."
+		"you have fighters in your bays? We have bays in our fighters."
+
+phrase "friendly civilian"
+	word
+		"Somebody"
+		"Some nut"
+	word
+		" "
+	word
+		"put"
+		"placed"
+	word
+		" "
+	word
+		"a blue phonebooth"
+		"a silver car"
+	word
+		" "
+	word
+		"on my bridge"
+		"in my armoury"
+		"in my cargohold"
+	word
+		" and "
+	word
+		"claimed to be"
+	word
+		" "
+	word
+		"a time traveller"
+		"from another time"
+	word
+		", and suddenly they disappeared."
+		", but the only travel he did was to a mental institution."
+		". I still don't know how she got into my spaceship."
+		". The thing looked bigger on the inside than on the outside."
+
+phrase "friendly civilian"
+	word
+		"I bought "
+	word
+		"a power generator"
+		"explosives"
+		"my new thrusters"
+		"some toothpaste"
+	word
+		" "
+	word
+		"from"
+	word
+		" "
+	word
+		"the ACME company"
+		"the Astronomical Conglomeration of Manufacturing Entities"
+	word
+		", a company many times smaller than the Syndicate."
+		", a minor company but with much better service than the Syndicate."
+		", a pretty insignificant company located just on one planet."
+		" but next time I'll buy from the Syndicate again."
```

Comment:
```suggestion
		", but next time, I'll buy from the Syndicate again."
```
OR
```suggestion
		", but next time I'll buy from the Syndicate again."
```
`next time, I'll buy from the Syndicate again` is an independent clause, so it needs a comma. `next time` is an introductory phrase and _technically_ needs a comma, but this is a spot it could slide - you could leave it out if you think it'd sound better without it. Your choice.